GNUmakefile

```
@@ -119,7 +119,7 @@ endif
 export DBUILD_ARGS=--build-arg DBUILD_DATE=${DBUILD_DATE} --build-arg DBUILD_REPO_URL=${DBUILD_REPO_URL} --build-arg DBUILD_SITE_URL=${DBUILD_SITE_URL} --build-arg ARCH=${ARCH}
 
 # Specify the name of cstor-base image
-CSTOR_BASE_IMAGE= ${IMAGE_ORG}/cstor-base:${BASE_TAG}
+CSTOR_BASE_IMAGE_AMD64= ${IMAGE_ORG}/cstor-base-amd64:${BASE_TAG}
 export CSTOR_BASE_IMAGE
```

Are we using base image with amd64 suffix?, or using the {ARCH-SUFFIX}?

q: This variable will be used only by travis build process . right?

```suggestion
export CSTOR_BASE_IMAGE_AMD64
```

---

Yes only amd64 and only travis builds will use it